src/wxMaxima.cpp

```diff
@@ -2641,7 +2641,7 @@ void wxMaxima::ReadVariables(wxString &data)
             if(name == "gnuplot_command")
             {
               m_gnuplotcommand = value;
-              wxLogMessage(wxString::Format(_("Gnuplot can be found at %s"),value.utf8_str()));
```

Comment:
This no more is the right place for a message, or the right message. But a wxLogMessage telling which gnuplot binary we have decided upon and why we did this might prove handy, one day: Any decision the program might make might be wrong and it is always good to document such places to make the search for bugs easier.